### DISCUSSION TITLE
Exportify working with Spotify song IDs?

### DISCUSSION THREAD

**Discussion by JBPROMO** [watsonbox/exportify#185] [General]
One of the great features in Exportify is (or was) that you can select a song ID of many songs, hit cntrl A select the ids and paste into a blank playlist on Spotify.  This made a lot of sense working with lots of playlists and exporting the actual full playlists into Spotify in just two clicks.  Or the feature is that you can sort on any field in Google sheets and select all of a playlist's songs by ID and paste into a new list.  This feature doesn't seem to be working.  Is this a bug within Exportify or Spotify just changed something?

**Comment by watsonbox** [comment]
I just tried it, and it still works fine for me with the Spotify desktop client (Mac).
